Nodejs/Tests/Core/Debugger/Commands/SetBreakpointCommandTests.cs

```diff
@@ -114,8 +147,8 @@ public class SetBreakpointCommandTests {
             Assert.AreEqual(commandId, setBreakpointCommand.Id);
             Assert.AreEqual(
                 string.Format(
-                    "{{\"command\":\"setbreakpoint\",\"seq\":{0},\"type\":\"request\",\"arguments\":{{\"line\":{1},\"column\":{2},\"type\":\"script\",\"target\":\"{3}\",\"ignoreCount\":1}}}}",
-                    commandId, line, column, fileName.Replace(@"\", @"\\")),
+                    "{{\"command\":\"setbreakpoint\",\"seq\":{0},\"type\":\"request\",\"arguments\":{{\"line\":{1},\"column\":{2},\"type\":\"scriptRegExp\",\"target\":\"{3}\",\"ignoreCount\":1}}}}",
+                    commandId, line, column, SetBreakpointCommand.CreateLocalScriptRegExp(fileName).Replace(@"\", @"\\")),
                 setBreakpointCommand.ToString());
         }
```

Comment:
If you haven't already - can you also take a look through existing UI tests and see if we need to change anything there?